### PR TITLE
Allow specifying the initial screen in secure conversations

### PR DESCRIPTION
**Pull request by gersonnoboa**
It is important that by default, welcome screen should be started. Integrator should do nothing special to start welcome screen, and has to manually opt for the chat transcript in order to go there. Please note the public enum and suggest better names if needed.

MOB-1883

### CODE REVIEW

File: GliaWidgets/Public/Glia/Glia.swift [hunk 1]
```diff
@@ -12,7 +12,15 @@ public enum EngagementKind: Equatable {
     /// Video call
     case videoCall
     /// Secure conversations
-    case messaging
+    case messaging(MessagingInitialScreen = .welcome)
+}
+
+/// The initial screen seen by a visitor when starting a secure conversation.
+public enum MessagingInitialScreen: Equatable {
```
[the remark this answers is not in the document]
Comment (gersonnoboa) [reply GliaWidgets/Public/Glia/Glia.swift:19]: Yes, it is correct, and yes it also makes sense to nest it inside of EngagementKind

[the remark this answers is not in the document]
Comment (gersonnoboa) [reply GliaWidgets/Public/Glia/Glia.swift]: Based on our conversation, changed this to `SecureConversations.InitialScreen` 09b920d7c3b7668064089fb56eee8c2c07604872

**Review by yurii-glia — APPROVED** [review]

**Comment by yurii-glia** [comment]
looks like after the latest changes there appeared Swiftlint vertical_whitespace issue.

File: GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift [hunk 2]
```diff
@@ -139,7 +149,7 @@ extension SecureConversations {
                     with: conf, cancelled: cancelled
                 )
             case .transcriptRequested:
-                navigateToTranscript()
+                _ = navigateToTranscript()
```
Comment (EgorovEI) [inline GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift:152]: You can use `@discardableResult`

**Review by EgorovEI — APPROVED** [review]

File: GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift (hunk 2 above)
Comment (gersonnoboa) [reply GliaWidgets/SecureConversations/SecureConversations.Coordinator.swift]: Fixed [here](https://github.com/salemove/ios-sdk-widgets/pull/507/files#diff-6a8aade246430e27807b109ce4a24745400410668b8ce4eb1c0477eeea0e14c6R239)

**Comment by gersonnoboa** [comment]
!merge

**Pull request merged**